src/refactor_nrepl/find/find_macros.clj

```diff
@@ -181,7 +186,11 @@
              (node->occurrence path macro-name offset zip-node))))
   zip-node)
 
-(defn- find-usages-in-file [macro ^File path]
+(defn- fully-qualified-name? [fully-qualified-name]
```

Comment:
This is generally useful and should probably be in `core`, but the parameter should be called `sym` or something as it's not known if the symbol is fully qualified or not.